src/utils/deepObjectProps.js

```diff
@@ -0,0 +1,43 @@
+const pathToArray = (...paths) => {
+	let keys;
+	try {
+		paths.forEach((path) => {
+			let newKeys = [];
+			if (typeof path === 'string') {
+				newKeys = path.split('.'); // for dot notation y.asd.asdasd.sdada
+			} else if (Array.isArray(path)) {
+				newKeys = path;
+			//	} else if (typeof path === 'object') {
+			//		// TODO: ?
+			} else {
+				throw new Error('Wrong type is adding as path in deepObjectProps.');
+			}
+			keys = [...keys, ...newKeys];
+		});
+		return keys;
+	} catch (err) {
+		// TODO: log it?
+		return '';
+	}
+};
+
+const get = (obj, ...paths) => { // TODO: write tests! 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4646529)